install/install.php

```diff
@@ -448,8 +448,6 @@ function step8()
 
 function update1($dbname)
 {
-    $_SESSION['is_installing'] = false;
```

Comment:
All checks are based on `isset($_SESSION['is_installing'])`, so setting it to false at this step is useless, and removing it from the session is useless too.